### PR TITLE
fix(organisation): refactor pagination

### PR DESCRIPTION
**Pull request by developer-sidani**
pagination and sorting from the query

### CODE REVIEW

**Comment by developer-sidani** [comment]
for now the sorting is done for the members, waiting the created at to be done

**Comment by developer-sidani** [comment]
as for the search it is also showing the button, becuase later the search is going to be from the same query

**Comment by developer-sidani** [comment]
Future plans for the organization list: when the button is pressed to show more, show the cached data same as displayed, and the new ones loading. Instead showing all organizations loading

**Comment by developer-sidani** [comment]
Just updated the code when the new data are loading! For now I am getting data 1 by one to show you how are the data fetched

**Comment by developer-sidani** [comment]
@5-mark please pin me when you approve this version so that I could update the organization to displaying 15 on default instead of 2, and getting 30 instead of 1 on button click

**Review by 5-mark — APPROVED** [review]
✅  works good

**Review by DarkNebula0 — CHANGES_REQUESTED** [review]

File: pkg/app/src/components/OrganisationCard/itemList.tsx [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
-					{items?.map((item, index) => (
-						<Grid item xs={4} key={index}>
-							<TileCard item={item} />
+			{allData?.map((item, index) => (
+				<Grid item xs={4} key={index}>
+					<TileCard item={item.node} />
+				</Grid>
+			))}
+
+			{loading
```
[the remark this answers is not in the document]
Comment (developer-sidani) [reply pkg/app/src/components/OrganisationCard/itemList.tsx:29]: noted

File: pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx [hunk 2]
```diff
@@ -17,9 +25,23 @@ const useStyles = makeStyles({
 			borderRadius: 8,
 		},
 	},
+	dropdownStyle: {
+		paddingTop: 3,
+		borderRadius: '7%',
+		backgroundColor: '#212B36',
```
[the remark this answers is not in the document]
Comment (developer-sidani) [reply pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx:31]: ok will look into it

File: pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx [hunk 3]
```diff
@@ -17,9 +26,20 @@ const useStyles = makeStyles({
...
+	dropdownStyle: {
+		paddingTop: 3,
+		borderRadius: '7%',
+		backgroundColor: PRIMARY_COLOR[3].background,
+		fontSize: 14,
+		'& .MuiMenuItem-root': {
+			paddingBottom: 20,
+			fontSize: 14,
+			minWidth: 200,
+			color: PRIMARY_COLOR[3].text,
```
Comment (hadi-ibra) [inline pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx:38]: @DarkNebula0 Can you give me some feedback regarding the way I implemented the global theme and whether there is a better way

**Comment by 2075** [comment]
- please use the edit function, if you want to add additional text to your comment vs posting n comments 👍 @developer-sidani 
- please resolve the conflicts
- regarding cards and other artefacts please stick to the ux provided and use template keys vs hardcoding any styling

File: pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx (hunk 3 above)
Comment (2075) [reply pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx:38]: you either 
- use template keys, whenever you have a customisation which is only needed once
- create a custom component with styling in components or
- globally change the styling of a component in the template overrides

usually you would have a global override base style and extend this locally based on your individual case.
hth

File: pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx [hunk 4]
```diff
@@ -17,9 +25,23 @@ const useStyles = makeStyles({
...
+		paddingTop: 3,
+		borderRadius: '7%',
+		backgroundColor: '#212B36',
+		fontSize: 14,
+		color: '#919EAB',
+
+		'& .MuiMenuItem-root': {
+			paddingBottom: 20,
+			fontSize: 14,
+			width: 200,
```
[the remark this answers is not in the document]
Comment (2075) [reply pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx]: this also belongs to template if it is a global requirement.

File: pkg/app/src/components/OrganisationCard/itemList.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (2075) [reply pkg/app/src/components/OrganisationCard/itemList.tsx]: omg

File: pkg/app/src/components/OrganisationCard/modules/sortOptionsTab.tsx [hunk 5]
```diff
@@ -1,42 +1,37 @@
...
-const sortOptions = [
-	{ value: 'hi-low', name: 'Member: High-Low' },
-	{ value: 'featured', name: 'Featured' },
-	{ value: 'newest', name: 'Newest' },
-]
-const SortOptionsTab = ({ classes }) => {
-	const [sortOption, setSortOption] = useState(sortOptions[0].value)
-
+const SortOptionsTab = ({ classes, sortOption, sortOptions, setSortOption }) => {
 	const handleChange = (event: SelectChangeEvent) => {
```
[the remark this answers is not in the document]
Comment (2075) [reply pkg/app/src/components/OrganisationCard/modules/sortOptionsTab.tsx]: @developer-sidani status?

File: pkg/app/src/components/OrganisationCard/itemList.tsx [hunk 6]
```diff
@@ -1,34 +1,38 @@
...
+import React, { useEffect, useState } from 'react'
 
 interface ComponentProps {
-	items: Organization[]
+	items: OrganizationEdge[]
 	loading: boolean
 }
 
 export function ItemList({ items, loading }: ComponentProps) {
+	const [allData, setAllData] = useState<OrganizationEdge[]>(items)
```
[the remark this answers is not in the document]
Comment (2075) [reply pkg/app/src/components/OrganisationCard/itemList.tsx:13]: @developer-sidani status?

**Comment by 2075** [comment]
you need to resolve the active conversations to be able to get a review and merge

File: pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx (hunk 3 above)
Comment (2075) [reply pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx]: @developer-sidani when will you fix this?

Comment (hadi-ibra) [reply pkg/app/src/components/OrganisationCard/modules/filtersSection.tsx]: if you mean regarding the font size and border radius we will create fix them in another issue to move on

**Review by 2075 — APPROVED** [review]

**Pull request merged**